lineapy/plugins/jinja_templates/airflow_dag_PythonOperator.jinja

```diff
@@ -29,7 +29,7 @@ def dag_teardown():
 default_dag_args = {
     "owner": "{{ OWNER }}",
     "retries": {{ RETRIES }},
-    "start_date": {{ START_DATE }},
+    "start_date": {{ START_DATE }},{%- if (dag_params != '"params":{}') %}    {{dag_params}},{%- endif -%}
```

Comment:
endline after the comma for readability.

Reply:
Suggesting the following change:

```python
default_dag_args = {
    "owner": "{{ OWNER }}",
    "retries": {{ RETRIES }},
    "start_date": {{ START_DATE }},
{%- if (dag_params != '"params":{}') %}
    {{ dag_params }},
{%- endif %}
}
```

Reply:
thanks

Reply:
done done